Description: various systemd fixes
 Drop ExecStartPre as systemd-kmsg-syslogd.service service was removed long
 ago.
 Alias the service to syslog.service.
 Add a Documentation= field to the unit file
 Also set the service type to Type=notify in the systemd unit file.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Forwarded: not-needed
Last-Update: 2013-03-09

---

--- syslog-ng-3.3.5.orig/contrib/systemd/syslog-ng.service
+++ syslog-ng-3.3.5/contrib/systemd/syslog-ng.service
@@ -3,10 +3,10 @@ Description=System Logger Daemon
 
 [Service]
 Sockets=syslog.socket
-ExecStartPre=/bin/systemctl stop systemd-kmsg-syslogd.service
 ExecStart=/usr/sbin/syslog-ng -F
 ExecReload=/bin/kill -HUP $MAINPID
 StandardOutput=null
 
 [Install]
 WantedBy=multi-user.target
+Alias=syslog.service
